src/index.js

```diff
@@ -13,9 +13,7 @@ module.exports = class SentryPlugin {
 		this.projectSlug = options.project
 		this.apiKey = options.apiKey
 
-		this.releaseVersion = _.isFunction(options.release)
-			? options.release()
-			: options.release
+		this.releaseVersion = options.releaseVersion;
```

Comment:
You've re-named the option key here, so the tests fail. Could you change this back to `options.release`?